### DISCUSSION TITLE
Additional roles to Data Owner and Expert

### DISCUSSION THREAD

**Discussion by dbwrangler** [open-metadata/OpenMetadata#18044] [Ideas]
Right now open-metadata displays 'data owner' against data objects and additionally 'expert' for data domain, In my experience the biggest collaboration problem is data consumers identifying data stewards, since it is the data stewards who are the data SMEs and deal with BAU of data issues not the owner. Owners tend to be higher level roles covering a larger portfolio. Open-metadata can easily solve this problem. My idea:

For data objects (such as a customer_orders table), rather that displaying just the data owner, open-metadata should display first the data steward, data architect and then data owner (maybe collapsible under 'collaboration'?). I suggest this is the correct order but it should be configurable. 

For data domains, sub domains: I'm not sure what 'expert' means but I think data steward and data architect should be used here too for consistency and ease of use an operating model.

I hope that makes sense!

**Comment by Prajwal214** [comment]
@dbwrangler Can you open a feature request [here](https://github.com/open-metadata/OpenMetadata/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=)

**Reply by dbwrangler** [reply]
thanks - https://github.com/open-metadata/OpenMetadata/issues/18078
